client/js/clientLogic.js

```diff
@@ -0,0 +1,296 @@
+        console.log(error);
+      });
+  }
+
+  function showReviewsByOrganization(organization) {
+    restaurantsAPI.getOrganizationReviews(organization,
+      function(reviewsResponse) {
+        drawModule.createOrganizationReviewsTable(reviewsResponse);
+      },
+      function(error) {
+        alert('Cannot get user reviews for: ' + organization);
+        console.log('Error displaying reviews for the organization ' +
+          organization);
+        console.log(error);
+      });
+  }
+
+  function getMyReservations() {
+    var username = connectionsAPI.getUser().displayName;
+    if (username) {
+      showReservationsByUser(username);
+    }
+  }
+
+  function showReservationsByUser(username) {
+    restaurantsAPI.getUserReservations(username,
+      function(reservationsResponse) {
+        drawModule.createReservationsTable(reservationsResponse);
+      },
+      function(error) {
+        alert('Cannot get user reservations for: ' + username);
+        console.log('Error displaying reservations for the user ' + username);
+        console.log(error);
+      });
+  }
+
+  function showReservationsByOrganization(organization) {
+    restaurantsAPI.getOrganizationReservations(organization,
+      function(reservationsResponse) {
+        drawModule.createReservationsTable(reservationsResponse);
+      },
+      function(error) {
+        alert('Cannot get user reservations for: ' + organization);
+        console.log('Error displaying reservations for the organization ' +
+          organization);
+        console.log(error);
+      });
+  }
+
+  function createNewReview(name, rating, description) {
+    restaurantsAPI.createNewReview(name, rating, description,
+      drawModule.closePopUpWindow,
+      function(err) {
+        alert('Cannot add review');
+        console.log('Error creating a new review: ' + name + ' ' + rating +
+          ' ' + description);
+        console.log(err);
+      }
+    );
+  }
+
+  function updateReview(reviewId, rating, description) {
+    restaurantsAPI.updateReview(reviewId, rating, description,
+      drawModule.closePopUpWindow,
+      function(err) {
+        alert('Cannot update review');
+        console.log('Error trying to update the review :' + reviewId +
+          ' with:' + rating + ' ' + description);
+        console.log(err);
+      }
+    );
+  }
+
+  function showReview(reviewId) {
+    restaurantsAPI.getReview(reviewId, function(reviewResponse) {
+      var restaurantReviewed =
+        JSON.parse(reviewResponse)[0].itemReviewed.name;
+      var reviewDiv = drawModule.createViewReviewDiv(reviewResponse);
+      drawModule.setPopupTitle('Review for ' + restaurantReviewed);
+      drawModule.setPopupContent(reviewDiv);
+      drawModule.openPopUpWindow();
+    },
+    function(err) {
+      alert('Cannot get the specified review');
+      console.log('Error retrieving the review: ' + reviewId);
+      console.log(err);
+    });
+
```

Comment:
I'm still seeing console.log where console.error , please could you check that you address all comments? 

thanks